dev/com.ibm.ws.kernel.boot.core/src/com/ibm/ws/kernel/boot/internal/BootstrapDefaults.java

```diff
@@ -0,0 +1,146 @@
+/*******************************************************************************
+ * Copyright (c) 2017 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.kernel.boot.internal;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.util.Locale;
+import java.util.Properties;
+
+import com.ibm.ws.kernel.boot.BootstrapConfig;
+
+/**
+ * Wrapper around the Liberty defaults file (in wlp/lib/platform/defaults).
+ */
+public class BootstrapDefaults {
+    /** Manifest header for the default kernel version */
+    static final String MANIFEST_KERNEL = "WebSphere-DefaultKernel";
```

Comment:
While the property keys look like manifest headers we no longer read them as manifest headers.  Should update the javadoc comment to indicate this is a property key read out of the defaults file